frontend/src/views/Transactions/AddEditViewTransaction.jsx

```diff
@@ -54,82 +53,94 @@ export const AddEditViewTransaction = () => {
   const mode = matches[matches.length - 1]?.handle?.mode
   const { transactionId } = useParams()
   const { data: currentUser, hasRoles, hasAnyRole } = useCurrentUser()
-  const isGovernmentUser = currentUser?.isGovernmentUser
   const [steps, setSteps] = useState(['Draft', 'Recommended', 'Approved'])
   const alertRef = useRef()
   const [alertMessage, setAlertMessage] = useState('')
   const [alertSeverity, setAlertSeverity] = useState('info')
+<<<<<<< feat/alex-return-txn-to-analyst-382
```

Comment:
Could you please resolve the conflicts? There are unresolved merge conflicts in the branch that need to be fixed before I can proceed with testing. Thank you